src/pages/signin/SignInPage.js

```diff
@@ -59,23 +74,37 @@ const defaultProps = {
 /**
  * @param {Boolean} hasLogin
  * @param {Boolean} hasValidateCode
+ * @param {Boolean} hasAccount
  * @param {Boolean} isPrimaryLogin
  * @param {Boolean} isAccountValidated
  * @param {Boolean} hasEmailDeliveryFailure
  * @returns {Object}
  */
-function getRenderOptions({hasLogin, hasValidateCode, hasAccount, isPrimaryLogin, isAccountValidated, hasEmailDeliveryFailure}) {
+function getRenderOptions({hasLogin, hasValidateCode, hasAccount, isPrimaryLogin, isAccountValidated, isSAMLEnabled, isSAMLRequired, isUsingSAMLLogin, hasEmailDeliveryFailure}) {
```

Comment:
Should we add these params in JSDoc comment?

Reply:
Yes 🫣